analyzers/src/SonarAnalyzer.CSharp/Rules/AvoidExcessiveClassCoupling.cs

```diff
@@ -80,22 +76,23 @@ public sealed class AvoidExcessiveClassCoupling : ParameterLoadingDiagnosticAnal
                 KnownType.System_Func_T1_T2_T3_T4_TResult,
                 KnownType.System_Lazy);
 
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(Rule);
```

Comment:
Line 43: Should be single line (it's a property)

Reply:
Line 104: Can be single line (broken for no reason)

Reply:
Line 274: I kept it on two lines but placed the && in front of the second.